### ISSUE TITLE
(Un-)publishing a normal pages gives spews warning message on console re: app url changes

### ISSUE THREAD

**Issue by mkoistinen** [django-cms/django-cms#2911]
Latest dev. 3.0b4.dev14

Simply publishing a page or unpublishing it spews this into the console (on runserver):

“Application url changed and urls_need_reloading signal fired. Please reload the urls.py or restart the server”

I thought this was for app-hooks? Is it for regular pages now?

**Comment by vinitkumar** [comment]
I still get this error on using aldyrn-newblog with django-cms: 3.0.12

```sh
GET /static/fonts/ubuntu.woff HTTP/1.1" 304 0
GET /en/admin/cms/page/28/en/publish/ HTTP/1.1" 302 0
Application url changed and urls_need_reloading signal fired. Please reload the urls.py or restart the ser
GET /en/blog/?edit_off HTTP/1.1" 200 21989
```

**Comment by yakky** [comment]
@vinitkumar is the page 28 the page the blog is attached to? it looks so

**Comment by vinitkumar** [comment]
Yes. It is.

**Comment by yakky** [comment]
So that's correct message. When you publish a page with an apphook on it, you must restart the instance or anyway reload ROOT_URLCONF

**Comment by vinitkumar** [comment]
@yakky Uploaded the gif. Please take a look.

**Comment by yakky** [comment]
@vinitkumar apparently this depends on setting up the correct template inheritance in the blog

**Comment by vinitkumar** [comment]
@yakky I am sorry, I didn't get you. How would I do that?

**Comment by yakky** [comment]
@vinitkumar your issue is unrelated and fixed in aldryn/aldryn-newsblog#169

**Comment by vinitkumar** [comment]
@yakky Thanks.
